Tests/app/fixtures/ExpectedDocuments/ExpectedDocuments.php

```diff
@@ -0,0 +1,566 @@
+<?php
+/*
+* This file is part of the ONGR package.
+*
+* (c) NFQ Technologies UAB <info@nfq.com>
+*
+* For the full copyright and license information, please view the LICENSE
+* file that was distributed with this source code.
+*/
+
+namespace ONGR\MagentoConnectorBundle\Tests\app\fixtures\ExpectedDocuments;
+
+use ONGR\MagentoConnectorBundle\Tests\app\fixtures\MagentoTestBundle\Document\Product;
+
+/**
+ * This class return tests Product document.
+ */
+class ExpectedDocuments
```

Comment:
I agree with decoupling chunks of data from the tests and even reusing test cases.